rllib/agents/dqn/r2d2.py

```diff
@@ -75,12 +75,6 @@
 
         # Update the target network every `target_network_update_freq` steps.
         "target_network_update_freq": 2500,
-
-        # Experimental flag.
```

Comment:
I'm assuming this just inherits from the dqn training iteration fn

Reply:
correct. R2D2 does not have its own `training_iteration` fn.